src/index.html

```diff
@@ -11,7 +11,184 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname" class="field">
+          Surname:
+          <input
+            type="text"
+            id="surname"
+            name="attribute"
+          >
+        </label>
+
+        <label for="name" class="field">
+          Name:
+          <input
+            type="text"
+            id="name"
+            name="attribute"
+          >
+        </label>
+
+        <label for="age" class="field">
+          How old are You?
+          <input
+            type="number"
+            id="age"
+            name="attribute"
+            min="12"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label for="birth" class="field">
+          Full date of birth:
+          <input
+            type="date"
+            id="birth"
+            name="attribute"
+          >
+        </label>
+
+        <label for="terms" class="field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="terms"
+            name="attribute"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label for="email" class="field">
+          E-mail:
+          <input
+            type="email"
+            id="email"
+            name="attribute"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label for="password" class="field">
+          Password:
+          <input
+            type="password"
+            id="password"
+            name="attribute"
+            minlength="5"
+            maxlength="32"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="field">
+          Do you love cats?
+
+          <label for="cat-yes">
+            <input
+              type="radio"
+              id="cat-yes"
+              name="cats"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label for="cat-no">
+            <input
+                type="radio"
+                id="cat-no"
+                name="cats"
+                value="no"
+            >
```

Comment:
```suggestion
            <input
              type="radio"
              id="cat-no"
              name="cats"
              value="no"
            >
```
Два пробела достаточно